Phonebook.Frontend/src/app/shared/components/user/user-detail/user-detail.component.html

```diff
@@ -255,7 +269,15 @@ <h4 mat-line>
               <ng-container i18n="@@ColumnTitleOrgUnit">
                 Organization Unit
               </ng-container>
-              <a mat-button [routerLink]="organigramLink">
+              <a
```

Comment:
revert

Reply:
It gets automatically formatted by the prettier bot

Reply:
I am talking about the `[routerLink]="organigramLink"`

Reply:
That's from the merge

Reply:
Yep but it's a change that should not be there.